HW_module_6.py

```diff
@@ -0,0 +1,155 @@
+                        break
+                    except FileNotFoundError:
+                        print('No file with such name. Try again')
+                    except PermissionError:
+                        print('No file with such name. Try again')
+            elif answer == 2:
+                while True:
+                    try:
+                        change_path = input('Enter the file path: ')
+                        if not os.path.isdir(change_path):
+                            os.mkdir(change_path)
+                        os.chdir(change_path)
+                        print(f'Now you directory is "{os.getcwd()}"')
+                        break
+                    except SyntaxError:
+                        print('You made a mistake. Try again')
+                    except FileNotFoundError:
+                        print('You made a mistake. Try again')
+                while True:
+                    try:
+                        file_name = input('Enter the file name with its format: ')
+                        with open(file_name, "r", encoding='utf-8') as source:
+                            f_contents = source.readlines()
+                        print('Such file exists')
+                        path_for_remove = os.path.join(str(change_path), str(file_name))
+                        os.chdir(original_path)
+                        is_true = False
+                        break
+                    except FileNotFoundError:
+                        print('No file with such name. Try again')
+                    except PermissionError:
+                        print('No file with such name. Try again')
+            else:
+                print('Try again here')
+        return f_contents, path_for_remove
+
+
+# ask a user what data he wants to print and then call a class and insert the data into file using inserting method
+if __name__ == "__main__":
+    while True:
+        print('Please enter your choice:', '1 - News', '2 - Private Ad', '3 - Divination','4 - Add data from another file', '5 - Nothing', sep='\n')
+        flag = input('Choose the appropriate number: ')
+        if flag == '1':
+            news = News(input('Please enter news text\n'),
+                        input('Please enter location\n'))
+            news_mess = news
+            news_mess.news_message()
+        elif flag == '2':
+            advng = Advertising(input('Please enter advertisement text\n'),
+                                input('Please enter expire date in the format dd/mm/yy\n'))
+            adv_message = advng
+            adv_message.advertising()
+        elif flag == '3':
+            guessing = Guess(input('Ask me about your near future\n'))
+            question_divination = guessing
+            question_divination.ask_future()
+        elif flag == '4':
+            f_contents, path_for_remove = FromAnotherSource().read_file()
+            with open("Module6.txt", "a", encoding='utf-8') as file:
```

Comment:
I expect that you will write the news feed in the same file which you're using in the print class:
Module6_paste.txt